src/Helpers/Click/ClickedArtistSeriesGroup.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ActionType,
+  ClickedArtistSeriesGroup,
+  ContextModule,
+  OwnerType,
+  PageOwnerType,
+} from "../../Schema"
+
+export interface ClickedArtistSeriesGroupArgs {
+  contextModule: ContextModule
```

Comment:
There aren't any commas in this object to separate each line are they
 necessary?